VideoLocker/src/main/java/org/edx/mobile/view/NavigationFragment.java

```diff
@@ -224,6 +226,10 @@ public void onClick(View v) {
             if (profile.email != null) {
                 drawerNavigationBinding.emailTv.setText(profile.email);
             }
+            Map map = new HashMap<>();
```

Comment:
I get a warning on the next few lines due to lack of types. Should be something like:
`Map<String,CharSequence> map = new HashMap<>();`